README.md

```diff
@@ -107,23 +107,30 @@ make
 
 GTDynamics now supports a Pybind11-based Python API.
 
-To start, please download and install the [GTwrap repository](https://github.com/borglab/wrap).
+GTWrap comes with the installation with GTSAM, which can also be used for generate the python bindings for GTDynamics.
 
 To compile and install the GTDynamics python library:
 
-1. In the build directory, run `cmake` with the flag `GTDYNAMICS_BUILD_PYTHON=ON`.
+1. Ensure that GTSAM is built with generated python bindings. If not, go to the build directory and run `cmake` with the flag `GTSAM_BUILD_PYTHON=ON`. Afterwards, install the GTSAM python package.
+
+    ```sh
+    cmake -DGTSAM_BUILD_PYTHON=ON ..
+    make && make python-install
+    ```
+
+2. In the GTDynamics build directory, run `cmake` with the flag `GTDYNAMICS_BUILD_PYTHON=ON`.
```

Comment:
I think you should explain that cmake in GTSAM and GTDynamics should ideally have the same, *non* /user/local prefix, and show the command with the corresponding prefix flag.

Reply:
Also this should not be bullet point 2, since a user can choose to do a `apt install libgtsam` and install gtwrap separately, in which case this is point 1.

GTWrap instructions (which includes installation via GTSAM) and GTDynamics instructions should be separate subsections.